accounts/views.py

```diff
@@ -91,9 +91,13 @@ def view_profile(request):
         )
         p_form = ProfileUpdateForm(instance=request.user.profile)
     current_user = request.user
+    is_admin = False
```

Comment:
I don't think this is a good way to check if a user is admin or not. The superusers can have different account with different names. It's better to check if the current user is a superuser.

Reference:
https://stackoverflow.com/questions/65421561/how-can-i-check-if-an-user-is-superuser-in-django

Reply:
Good point, change applied!